integration

```diff
@@ -160,6 +160,7 @@
   "bremor/public_transport_victoria",
   "brianegge/home-assistant-sdnotify",
   "briis/weatherflow_forecast",
+  "brmccrary/nuvo_simple",
```

Comment:
The current release has an invalid file <https://github.com/brmccrary/nuvo_simple/blob/v1.3.1/hacs.json>